<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="leftLabel">
        <property name="text">
         <string>Links</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rightLabel">
        <property name="text">
         <string>Rechts</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Zählweise</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="nothingRadioButton">
         <property name="text">
          <string>Nichts zählt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="leftRadioButton">
         <property name="text">
          <string>Links zählt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rightRadioButton">
         <property name="text">
          <string>Rechts zählt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="bothRadioButton">
         <property name="text">
          <string>Beides zählt</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="triggerButton">
      <property name="text">
       <string>Zählen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>Beenden</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
